squid-mixin/dashboards/squid-overview.libsonnet

```
@@ -0,0 +1,1454 @@
+      'rate(squid_server_http_kbytes_in_kbytes_total{job=~"$job", instance=~"$instance"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='HTTP',
+    ),
+    prometheus.target(
+      'rate(squid_server_other_kbytes_in_kbytes_total{job=~"$job", instance=~"$instance"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='other',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Server received throughput',
+  description: 'The number of HTTP, FTP, and other server throughput.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'KBs',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local cacheLogsPanel = {
+  datasource: lokiDatasource,
+  targets: [
+    {
+      datasource: lokiDatasource,
+      editorMode: 'code',
+      expr: '{filename="/var/log/squid/cache.log", job=~"$job"} |= ``',
+      queryType: 'range',
+      refId: 'A',
+    },
+  ],
+  type: 'logs',
+  title: 'Cache logs',
+  description: 'The cache.log file that contains the debug and error messages that Squid generates.',
+  options: {
+    dedupStrategy: 'none',
+    enableLogDetails: true,
+    prettifyLogMessage: false,
+    showCommonLabels: false,
+    showLabels: false,
+    showTime: false,
+    sortOrder: 'Descending',
+    wrapLogMessage: false,
+  },
+};
+
+local accessLogsPanel = {
+  datasource: lokiDatasource,
+  targets: [
+    {
+      datasource: lokiDatasource,
+      editorMode: 'code',
+      expr: '{filename="/var/log/squid/access.log", job=~"$job"} |= ``',
+      queryType: 'range',
+      refId: 'A',
+    },
+  ],
+  type: 'logs',
+  title: 'Access logs',
+  description: 'The access.log file contains a record of all HTTP requests and responses processed by the Squid proxy server.',
```

```suggestion
  description: 'The access log file contains a record of all HTTP requests and responses processed by the Squid proxy server.',
```

Nit: Could this potentially be renamed and reconfigured? May just want to use a non-filename dependent description.